hooks/waitForBooting.sh

```diff
@@ -1,11 +1,16 @@
-
-
+#!/bin/sh
 
 echo "====> Wait for Boot Multi user"
-if bash vbox.sh waitForText $VM_OS_NAME "Boot Multi user" 10 ; then
-  echo "====> OK, enter"
-  bash vbox.sh enter
-fi
-
-
+try_counter=0
+while ! bash vbox.sh waitForText "$VM_OS_NAME" "Boot Multi user" 10 ; do
+  if [ "$try_counter" -ge 10 ] ; then
+    echo "====> Too many tries... exiting."
+    exit 1
```

Comment:
doesn't need to "exit" here.
If we don't find the booting menu, it's ok to just continue.